go.mod

```diff
@@ -1,8 +1,15 @@
-module github.com/maxsupermanhd/FactoCord-3.0
+module github.com/maxsupermanhd/FactoCord-3.0/v3
 
-go 1.14
+go 1.22
```

Comment:
I'll admit this is overly restrictive, we could drop it to Go 1.21 if needed.

Reply:
Yes, please set it to the lowest version required by the dependencies